docs/data/charts/scatter/scatter.md

```diff
@@ -116,3 +116,13 @@ See [Axis—Grid](/x/react-charts/axis/#grid) documentation for more information
 ### Shape 🚧
 
 ### Size 🚧
+
+## Plot Customization
+
+You can customize the plotting of the data in a scatter chart by providing custom components as `children` of the `ScatterChart` component.
+
+A scatter chart's series can be accessed through the `useScatterSeries` hook. This hook returns the order of the series and information about the series themselves, including their data points, color, etc.
```

Comment:
We break lines after each sentence, because the smallest unit of GitHub suggestion is the line. SO the more line you get the more precise you can be in your review. And markdown consider a single line break as a space.

```suggestion
A scatter chart's series can be accessed through the `useScatterSeries` hook.
This hook returns the order of the series and information about the series themselves, including their data points, color, etc.
```

Reply:
Makes sense, will do that from now on 👍